pkg/skaffold/k8sjob/logger/log.go

```diff
@@ -164,13 +164,26 @@ func (l *Logger) streamLogsFromKubernetesJob(ctx context.Context, id, namespace
 				return false, nil
 			}
 
-			for event := range w.ResultChan() {
-				pod, ok := event.Object.(*corev1.Pod)
-				if ok {
-					podName = pod.Name
-					// TODO(aaron-prindle) add support for jobs w/ multiple pods in the future
-					break
+			done := make(chan bool)
+			go func() {
+				for event := range w.ResultChan() {
+					pod, ok := event.Object.(*corev1.Pod)
+					if ok {
+						podName = pod.Name
+						done <- true
+						break
+					}
 				}
+			}()
+
+			select {
+			case <-ctx.Done():
+				fmt.Println("receive loop ends")
```

Comment:
Do we want a log message here? Can we use `log.Entry(ctx).Debug` or any of its variations instead?

Reply:
Sorry, this was added unintentionally.  Deleted now